src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
```

Comment:
Rather than converting the HTMLCollection to an array with `Array.from()`, you could use a simple for loop to iterate over the rows. This would be more performant as it avoids creating a new array.

Reply:
In the first my code I used a simple for, but you recommended using modern array methods like forEach. Should I use my first code?